doc/classes/PackedByteArray.xml

```diff
@@ -151,6 +151,7 @@
 			<param index="0" name="byte_offset" type="int" />
 			<description>
 				Decodes a 64-bit unsigned integer number from the bytes starting at [param byte_offset]. Fails if the byte count is insufficient. Returns [code]0[/code] if a valid number can't be decoded.
+				[b]Warning:[/b] Since GDScript's [int] is a 64-bit signed integer, the result will be automatically cast to a signed value when ran in GDSCript.
```

Comment:
I think this should be more clear about the effects themselves, the fact that it is cast to a signed value is pretty irrelevant to most, but the specifics of how this affects the range is more important. And what this means and what it doesn't mean

Reply:
Added more details about the effect of the conversion.

Reply:
Will take a look tomorrow!